rules/okta/credential_access_suspicious_okta_user_password_reset_or_unlock_attempts.toml

```diff
@@ -0,0 +1,75 @@
+[metadata]
+creation_date = "2020/08/19"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of Okta user password reset or account unlock attempts. An adversary may attempt to obtain
+unauthorized access to an account Okta user account using these methods and attempt to blend in with normal activity in
+their target's environment and evade detection.
+"""
+from = "now-120m"
```

Comment:
Is 5 attempts over 2 hours too loose? Especially since it is for so many different types of attempts. A frustrated user may attempt multiple methods. Could we tighten this down to a shorter duration and or maybe more attempts?

Reply:
I'm open to that. What do you think about 5 attempts within an hour?

Reply:
I think that is better. Can you add a snippet in the `false_positives` recommending tuning that ratio (probably a good thing to do for all threshold rules?).

Reply:
```suggestion
from = "now-60m"
```

Reply:
> I think that is better. Can you add a snippet in the `false_positives` recommending tuning that ratio (probably a good thing to do for all threshold rules?).

@brokensound77 - I can add a note to the FP field. 👍

Am I correct in thinking that if a user wants to customize the schedule and threshold values, they'll need to duplicate the rule and set the values in the new rule. I don't think they can edit these values in our prebuilt rules.

Reply:
Added note to FP field and adjusted formatting of query, as per conversation with @brokensound77